pkg/netpol/connlist/connlist.go

```diff
@@ -47,15 +47,15 @@ type ConnlistAnalyzer struct {
 // The new interface
 // ConnlistFromResourceInfos returns the allowed-connections list from input slice of resource.Info objects,
 // and the list of all workloads from the parsed resources
```

Comment:
update this comment, explain when `[]ExposedPeer` is returned as not just `nil`, and what it represents.

Reply:
will be resolved in next PR